spec/lib/efile/md/md502_r_calculator_spec.rb

```diff
@@ -222,4 +222,114 @@
       expect(instance.lines[:MD502R_LINE_10B].value).to eq 70
     end
   end
+
+  describe "#calculate_line_11a" do
+    context "when primary qualifies" do
+      before do
+        allow_any_instance_of(StateFileMdIntake).to receive(:qualifies_for_pension_exclusion?).with(:primary).and_return(true)
+        allow_any_instance_of(Efile::Md::Md502RCalculator).to receive(:calculate_line_1a).and_return(100_000)
+        allow_any_instance_of(Efile::Md::Md502RCalculator).to receive(:calculate_line_10a).and_return(80_000)
+        allow_any_instance_of(Efile::Md::Md502RCalculator).to receive(:calculate_line_9a).and_return(line_9_a_value)
```

Comment:
oh i like the approach of just having one of the 2 values being compared tied to a variable!